src/components/ReviewDeposit.tsx

```diff
@@ -49,80 +55,99 @@ function ReviewDeposit({
     transactionDeadlineCustom,
   )
   const shouldDisplayGas = !!gasStandard
+
+  const DepositeInfoItem = styled(Box)(({ theme }) => ({
+    display: "flex",
+    minWidth: "100%",
+    marginBottom: theme.spacing(2),
+    "&>:first-child": {
```

Comment:
Can this flex solution be done on the component level and outside of explicit styling? Why is it that only the first and last childs are problematic with styles?

Reply:
There are several `DepositInfo`, so. i made this `component` to avoid to giving props repeatly.
And, by giving property first and last component, It doesn't need to props on the child component in `DepositInfoItem`.